packages/core/jkl/_spacing.scss

```diff
@@ -5,119 +5,132 @@
 
 /// Tilsvarer 104px
 /// @type Number
-$spacing-3xl: 6.5rem;
+$spacing-104: 6.5rem;
+$spacing-3xl: $spacing-104;
 
 /// Tilsvarer 168px
 /// @type Number
-$spacing-4xl: 10.5rem;
+$spacing-168: 10.5rem;
+$spacing-4xl: $spacing-168;
 
 /// Lar deg gjøre oppslag på navnet uten `spacing`-prefix for å få ut rem-verdien
 /// @type Map
-$spacing-new: (
-    "3xs": (
-        "small-screen": $spacing-3xs,
-        "large-screen": $spacing-3xs,
-    ),
-    "2xs": (
-        "small-screen": $spacing-2xs,
-        "large-screen": $spacing-2xs,
-    ),
-    "xs": (
-        "small-screen": $spacing-xs,
-        "large-screen": $spacing-xs,
-    ),
-    "s": (
-        "small-screen": $spacing-s,
-        "large-screen": $spacing-s,
-    ),
-    "m": (
-        "small-screen": $spacing-m,
-        "large-screen": $spacing-m,
-    ),
-    "l": (
-        "small-screen": $spacing-m,
-        "large-screen": $spacing-l,
-    ),
-    "xl": (
-        "small-screen": $spacing-l,
-        "large-screen": $spacing-xl,
-    ),
-    "2xl": (
-        "small-screen": $spacing-xl,
-        "large-screen": $spacing-2xl,
-    ),
-    "3xl": (
-        "small-screen": $spacing-2xl,
-        "large-screen": $spacing-3xl,
-    ),
-    "4xl": (
-        "small-screen": $spacing-3xl,
-        "large-screen": $spacing-4xl,
-    ),
+$spacing: (
+    "2": $spacing-2,
+    "4": $spacing-4,
+    "8": $spacing-8,
+    "12": $spacing-12,
+    "16": $spacing-16,
+    "24": $spacing-24,
+    "32": $spacing-32,
+    "40": $spacing-40,
+    "64": $spacing-64,
+    "104": $spacing-104,
+    "168": $spacing-168,
+    // Gamle t-skjortestørrelser for kompatibilitet
+    "3xs": $spacing-3xs,
+    "2xs": $spacing-2xs,
+    "xs": $spacing-xs,
+    "s": $spacing-s,
+    "m": $spacing-m,
+    "l": $spacing-l,
+    "xl": $spacing-xl,
+    "2xl": $spacing-2xl,
+    "3xl": $spacing-3xl,
+    "4xl": $spacing-4xl,
 );
 
-$positions: (top, bottom, left, right);
-
-@mixin spacing($spacing-name) {
-    // Separate spacing/position into spacing and position variables:
-    $start-index: string.index($spacing-name, "/") or 0;
-    $end-index: string.index($spacing-name, "/") or 9999999;
-    $spacing: string.slice($spacing-name, 0, $start-index - 1);
-    $position: string.slice($spacing-name, $end-index + 1);
+/// Her defineres par av spacinger for dynamiske spacing-klasser, som f.eks. .jkl-spacing-16-24
+/// @type Map
+$spacing-pairs: (("16", "24"), ("24", "40"), ("32", "40"), ("40", "64"), ("64", "104"), ("104", "168"));
 
-    @if map.has-key($spacing-new, $spacing) {
-        $variants: map.get($spacing-new, $spacing);
+/// Posisjoner som kan benyttes for spacing
+/// @type Map
+$positions: (top, bottom, left, right);
 
-        @if string.length($position) == 0 {
-            margin: map.get($variants, "small-screen");
+/// Gir deg marginer basert på stegene i spacing-skalaen.
+/// @param $steps Enten navnet på ett steg i skalaen, eller navnet på tosteg i skalaen adskilt av "/".
```

Comment:
Det er kjedelig å liste ut, men det hjelper code completion _veldig_ mye om vi kan få inn alle gyldige verdier her i steps på samme måte som for position. Se `text-style` i `_typography` for eksempel.

Reply:
Haha, var redd for at du ville spørre om dette 😅 Husk at man potensielt får `antall steg * antall steg` som mulige verdier her (en del færre i praksis siden man kun vil gå fra lite til stort steg). Kan godt skrive inn, men blir en _durabelig_ lang liste...

Reply:
Jeg vet, men det forbedrer DX med sånn cirka 1000%

Reply:
> Så lenge man har VSCode-pluginen

Pass på at det virker for IntelliJ også :laughing: